db/sql_stmt_cache.c

```diff
@@ -151,6 +151,10 @@ static void stmt_cache_free_entry(stmt_cache_entry_t *entry)
 
 static int stmt_cache_finalize_entry(stmt_cache_entry_t *entry)
```

Comment:
How about we drop this function and use `stmt_cache_finalize_entry_int` instead?

Reply:
Yes they look identical will drop this function and use that one instead.